content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -28,10 +28,8 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
      project path of the corresponding `.dvc` file.
    - If the model file is in remote storage and is not DVC-tracked, enter the
      absolute path of the model file.
-   - If you use [MLEM] to save your model, use the path to the binary file that
-     MLEM generates. After you have run
-     [`mlem init`](https://mlem.ai/doc/command-reference/init), Iterative Studio
-     will be able to parse the `.mlem` file to extract model metadata.
+   - If you use [MLEM] to save your model, use the path to the binary file or
+     folder that MLEM generates.
```

Comment:
Do we need this much text about what path to use? Seems like the options are either:
1. If the model file is in the Git repository (including if it is saved with DVC and/or MLEM), enter the relative path of the model (from the repository root).
2. Otherwise, enter the URL to the model file in the cloud.

Reply:
@aguschin Thoughts on this?

Reply:
Looks right to me. Fixing!